packages/client3/components/design-system/logo/logo.stories.tsx

```diff
@@ -1,12 +1,12 @@
 import { Story, Meta } from "@storybook/react";
 
-import { GoldfinchLogo } from ".";
+import { FreeArtistsLogo } from ".";
 
 export default {
   title: "Components/Logo",
```

Comment:
```suggestion
  title: "FAD/Components/Logo",
```

Reply:
> 

1. Got new logos from design to meet the dimensions. 

2. Fixed the setup directory. Apologizes looks like those two scripts were copied into the git folder

3. Changed fa-[logo] to FAD-logo as per group decision.